cli/src/main/kotlin/commands/EvaluatorCommand.kt

```diff
@@ -97,10 +97,11 @@ object EvaluatorCommand : CommandWithHelp() {
             return if (evaluator.checkSyntax(script)) 0 else 2
         }
 
-        val evaluatorRun by lazy { evaluator.run(script) }
+        val absoluteOutputDir = outputDir?.absoluteFile?.normalize()
```

Comment:
Sorry, I don't get the commit message. How does this fail earlier now? Earlier before what?

Reply:
It now fails before running the rules, which can take some time. I have update the commit message to make this more clear.

Reply:
But `evaluatorRun` is `lazy`, so why were the rules evaluator before the "one of the output files..." message?

Reply:
You are right, I tested again and it turns out the delay was caused by parsing the ORT result file. I have tested with a large result file yesterday which took about 10 seconds to parse. I have adapted the change accordingly.